client/src/app/core/core-services/active-meeting-id.service.ts

```diff
@@ -32,7 +32,7 @@ export class ActiveMeetingIdService {
             this.currentMeetingId = newMeetingId;
             this.meetingIdSubject.next(newMeetingId);
         }
-        if (Number.isNaN || (newMeetingId && newMeetingId !== this.currentMeetingId)) {
+        if (Number.isNaN(newMeetingId) || (newMeetingId && newMeetingId !== this.currentMeetingId)) {
```

Comment:
It would be nice, if `null` was used instead of NaN. Can you find the place where NaN is generated (I suspect a cast from string to integer). Check *there* if the cast was sucessfull and otherwise return null.